docs/docs/user/migration-guide/v6-v7.md

```diff
@@ -60,3 +60,30 @@ As a result of the restructure, some imports need to be adjusted:
 -components/__init__.py
 +kpops/components/custom/__init__.py
 ```
+
+## [Call destroy from inside of reset or clean](https://github.com/bakdata/kpops/pull/501)
+
+Before v7, the KPOps CLI executed `destroy` before running `reset/clean` to ensure the component was destroyed.
+
+This logic has changed. From now on, the `destroy` method should be called within the `reset/clean` method if needed by the user.
+
+During the migration to v7, you should check your custom components and see if they are overriding the `reset/clean` methods. If so, you would need to add the `destroy` method at the beginning of your `reset/clean` override.
+
+#### components.py
+
+In the following example, we must call the `destroy` method inside the `clean` method. ** Otherwise, the logic of destroy will not be executed!**
```

Comment:
one extra space 🧐 
```suggestion
In the following example, we must call the `destroy` method inside the `clean` method. **Otherwise, the logic of destroy will not be executed!**
```